android/expoview/build.gradle

```diff
@@ -214,9 +214,6 @@ dependencies {
   compileOnly 'org.glassfish:javax.annotation:3.1.1'
   api 'de.greenrobot:eventbus:2.4.0'
 
-  // Be careful when upgrading! Upgrading might break experience scoping. Check with Jesse. See Analytics.resetAmplitudeDatabaseHelper
-  compileOnly 'com.amplitude:android-sdk:2.9.2'
```

Comment:
I think this dependency needs to stay here (along with the app/build.gradle entry) because of Analytics.java which uses Amplitude for our own logging in the Expo client. Could you make sure the logic there still works as expected, especially the `resetAmplitudeDatabaseHelper` method referenced here?

Reply:
Thank you for noticing this! Fixed in https://github.com/expo/expo/pull/6174/commits/417222e2745ffb077f989c88cd2b5f93fcada1b9.